crates/indexer/src/geyser/programs/magic_eden_haus.rs

```diff
@@ -46,7 +49,8 @@ async fn process_execute_sale(
             id: None,
             buyer: Owned(accts[0].clone()),
             seller: Owned(accts[1].clone()),
-            auction_house: Owned(accts[9].clone()),
+            auction_house: None,
```

Comment:
Since ME is a fork of auction house I really do think one of the accounts is an auction house. It be a massive refactor to remove it and rather fruitless.